contentrepo/settings/base.py

```diff
@@ -13,7 +13,8 @@
 DEFAULT_SECRET_KEY = "please-change-me"
 SECRET_KEY = os.environ.get("SECRET_KEY") or DEFAULT_SECRET_KEY
 ALLOWED_HOSTS = env.list("ALLOWED_HOSTS", default=["localhost"])
-
+WAGTAILDOCS_EXTENSIONS = ["doc", "docx", "xls", "xlsx", "ppt", "pptx", "pdf", "txt"]
+WAGTAILDOCS_DOCUMENT_MODEL = "home.CustomDocument"
```

Comment:
Looking at the documents, it doesn't seem like we need a whole custom document model to be able to use the `WAGTAILDOCS_EXTENSIONS` configuration, have you tried it without the custom model?

I'm worried about existing documents and how that's handled with a migration over to a new model.

Reply:
This was the only way I could get File extension “exe” is not allowed. Allowed extensions are: doc, docx, xls, xlsx, ppt, pptx, pdf, txt or else it will show like this File extension “” is not allowed. Allowed extensions are: doc, docx, xls, xlsx, ppt, pptx, pdf, txt.

Reply:
I did a test on my local environment, and applying this change removes all documents, so that's unfortunately not something we can apply and merge, otherwise we'll lose all documents in our production instances.

Before trying to add a migration to migrate things over to the new document model, however, this looks to me like a bug in Wagtail. 

So what I would suggest here is, to create a new blank project, using the latest version of wagtail (you can follow https://docs.wagtail.org/en/stable/getting_started/tutorial.html), and add the minimal amount to be able to upload documents, and see if you can replicate this issue. If you can, then we should create a detailed bug report on the wagtail repo, with all the reproduction information, as this is then something that needs to be fixed upstream in wagtail.

Reply:
Will have a look.
Thanks